services/huaweicloud/src/main/java/org/eclipse/osc/orchestrator/plugin/huaweicloud/builders/terraform/TFExecutor.java

```diff
@@ -0,0 +1,121 @@
+package org.eclipse.osc.orchestrator.plugin.huaweicloud.builders.terraform;
+
+import lombok.extern.java.Log;
+import org.eclipse.osc.services.ocl.loader.Ocl;
+
+import java.io.*;
+import java.nio.file.FileSystems;
+import java.util.Arrays;
+
+@Log
+public class TFExecutor {
+
+    private final Ocl ocl;
+    private String tfPath;
+    private String workPath;
+
+    public TFExecutor(final Ocl ocl) {
+        this.ocl = ocl;
+    }
+
+    public boolean createWorkspace() {
+        File ws = new File("terraform_ws" + FileSystems.getDefault().getSeparator() + ocl.getName());
+
+        if (!ws.exists() && !ws.mkdirs()) {
+            log.warning("File path not created. " + ws.getAbsolutePath());
+            return false;
+        }
+
+        tfPath = ws.getAbsolutePath() + FileSystems.getDefault().getSeparator() + "main.tf";
+        String verPath = ws.getAbsolutePath() + FileSystems.getDefault().getSeparator() + "/version.tf";
+        workPath = ws.getAbsolutePath();
+        log.info("Terraform working directory is " + workPath);
+
+        try {
+            try (FileWriter versionFile = new FileWriter(verPath)) {
+                versionFile.write(""
+                    + "terraform {\n"
+                    + "  required_providers {\n"
+                    + "    huaweicloud = {\n"
+                    + "      source = \"huaweicloud/huaweicloud\"\n"
+                    + "      version = \">= 1.20.0\"\n"
+                    + "    }\n"
+                    + "  }\n"
+                    + "}");
+            }
+        } catch (IOException ex) {
+            log.warning(ex.getMessage() + Arrays.toString(ex.getStackTrace()));
+            return false;
+        }
+
+        return true;
+    }
+
+    public boolean createTFScript() {
+        Ocl2Hcl hcl = new Ocl2Hcl(ocl);
+        String hclStr = hcl.getHcl();
+
+        try {
+            try (FileWriter tfFile = new FileWriter(tfPath)) {
+                tfFile.write(hclStr);
+            }
+        } catch (IOException ex) {
+            log.warning(ex.getMessage() + Arrays.toString(ex.getStackTrace()));
```

Comment:
I think we should unify our log framework, instead of ex.getStackTrace(),  Just using log.warn("some message", ex)